packages/fluent/scss/chip/_variables.scss

```diff
@@ -0,0 +1,735 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+@use "../core/color-system/" as *;
+
+// Chip
+
+
+/// Width of the border around the button.
+/// @group chip
+$kendo-chip-border-width: 1px !default;
+/// The spacing between the text and the icons of the chip.
+/// @group chip
+$kendo-chip-spacing: map-get( $kendo-spacing, 1 ) !default;
+
+/// The chip avatar width.
+/// @group chip
+$kendo-chip-avatar-width: 1em;
+/// The chip avatar height.
+/// @group chip
+$kendo-chip-avatar-height: 1em;
+/// The chip avatar flex basis.
+/// @group chip
+$kendo-chip-avatar-flex-basis: 1em;
+/// The text color of disabled chip.
+/// @group chip
+$kendo-chip-disabled-text: get-theme-color-var( neutral-90 ) !default;
+/// The background of solid disabled chip.
+/// @group chip
+$kendo-chip-solid-disabled-bg: get-theme-color-var( neutral-20 ) !default;
+/// The border of solid disabled chip.
+/// @group chip
+$kendo-chip-solid-disabled-border: get-theme-color-var( neutral-20 ) !default;
+/// The background of outline disabled chip.
+/// @group chip
+$kendo-chip-outline-disabled-bg: $kendo-component-bg !default;
+/// The border of outline disabled chip.
+/// @group chip
+$kendo-chip-outline-disabled-border: get-theme-color-var( neutral-90 ) !default;
+
+/// Offset of the outline focused chip.
+/// @group chip
+$kendo-chip-focus-offset: map-get( $kendo-spacing, thin ) !default;
+/// Outline width of the outline focused button.
+/// @group chip
+$kendo-chip-focus-outline-width: map-get( $kendo-spacing, hair) !default;
+/// Outline style of the outline focused button.
+/// @group chip
+$kendo-chip-focus-outline-style: solid !default;
+
+/// Horizontal padding of the chip.
+/// @group chip
+$kendo-chip-padding-x: map-get( $kendo-spacing, 1 ) !default;
+$kendo-chip-padding-x-sm: map-get( $kendo-spacing, 1 ) !default;
+$kendo-chip-padding-x-md: map-get( $kendo-spacing, 1 ) !default;
+$kendo-chip-padding-x-lg: map-get( $kendo-spacing, 1 ) !default;
+
+/// Vertical padding of the chip.
+/// @group chip
+$kendo-chip-padding-y: map-get( $kendo-spacing, 1 ) !default;
+$kendo-chip-padding-y-sm: map-get( $kendo-spacing, 1 ) - map-get( $kendo-spacing, thin )  !default;
```

Comment:
I believe the vertical padding should be the same as in the Button component (I know the design shows a different thing but I believe that it is a problem with the design file):

SM: `map-get( $kendo-spacing, 1  )`
MD: `map-get( $kendo-spacing, 2 ) - map-get( $kendo-spacing, thin )`
LG: `map-get( $kendo-spacing, 2 )`